Formula/pinot.rb

```diff
@@ -24,7 +25,7 @@ def install
   end
 
   service do
-    run [opt_bin/"pinot-admin", "QuickStart", "-type", "BATCH", "-dataDir", var/"lib/pinot/data"]
+    run [opt_bin/"pinot-admin", "QuickStart", "-type", "DEFAULT", "-dataDir", var/"lib/pinot/data", "-zkAddress", "localhost:2181"]
```

Comment:
Why this particular port?

Reply:
This is the default port of Zookeeper, which has been added as a dependency.

Reply:
Did pinot not work before?

Reply:
The confusion I think here for anyone would be that the `QuickStart` command never used to be the way you would run a production deployment of Pinot (as of `0.9.3`). It was more like a playground for a user to evaluate the database locally. But as of `0.10.0` the `DEFAULT` quickstart recipe can be used for integration testing, evaluation, or even single node production deployments.

Reply:
Homebrew generally doesn't aim to be production ready though, we ship the easiest configuration for developers.